mdb.py

```
@@ -40,6 +43,279 @@ def get_config(config="config/config.yml"):
+            if isinstance(item, dict):
+                yaml_str += f"{indent_str}- "
+                yaml_str += dict_to_yaml_inline(item, indent)
+            else:
+                yaml_str += f"{indent_str}- {format_yaml_value(item)}\n"
+    else:
+        yaml_str = format_yaml_value(data)
+
+    return yaml_str
+
+
+def dict_to_yaml_inline(data, indent=0):
```

Ruff (0.15.4)

[warning] Unused function argument: `indent`

(ARG001)